src/framework/xr/xr-anchors.js

```diff
@@ -171,30 +200,23 @@ class XrAnchors extends EventHandler {
             callback = rotation;
 
             if (!this._supported) {
-                if (callback) callback(new Error('Anchors API is not supported'), null);
+                callback?.(new Error('Anchors API is not supported'), null);
                 return;
             }
 
             if (!hitResult.createAnchor) {
-                if (callback) callback(new Error('Creating Anchor from Hit Test is not supported'), null);
+                callback?.(new Error('Creating Anchor from Hit Test is not supported'), null);
                 return;
             }
 
             hitResult.createAnchor()
                 .then((xrAnchor) => {
-                    const anchor = new XrAnchor(this, xrAnchor);
-                    this._index.set(xrAnchor, anchor);
-                    this._list.push(anchor);
-
-                    if (callback)
-                        callback(null, anchor);
-
+                    const anchor = this._createAnchor(xrAnchor);
+                    callback?.(null, anchor);
                     this.fire('add', anchor);
                 })
                 .catch((ex) => {
-                    if (callback)
-                        callback(ex, null);
-
+                    callback?.(ex, null);
```

Comment:
you changed all these `if (callback) callback();` to `callback?.()` in this commit, but was that overwritten by merge from main? I don't see those changes in up to date branch.

Reply:
yep all good now, thanks!